packages/block-library/src/navigation/index.js

```diff
@@ -50,6 +52,14 @@ export const settings = {
 			},
 		],
 	},
+	__experimentalLabel( { ref } ) {
+		const { navigationMenu } = useNavigationMenu( ref );
+		return buildNavigationLabel(
+			navigationMenu?.title,
+			1, // This has to be 1 because in this context there is only one Navigation shown.
+			navigationMenu?.status
+		);
```

Comment:
Given this function signature is a little opaque (especially the 2nd argument) I wonder whether it's better just to pass the entire navigationMenu and have the function grab the bit it needs.

I feel perhaps the consumer needs to know too much.

What do you think?

Reply:
In that case maybe make the function accept an object with named arguments? It might be easier to understand.

Not a blocker though. Just a code quality idea.